pages/dao/[symbol]/proposal/components/instructions/Dual/StakingOption.tsx

```diff
@@ -0,0 +1,200 @@
+    return obj
+  }
+  useEffect(() => {
+    handleSetInstructions(
+      { governedAccount: governedAccount, getInstruction },
+      index
+    )
+  }, [form])
+  useEffect(() => {
+    setGovernedAccount(form.baseTreasury?.governance)
+  }, [form.baseTreasury])
+  const schema = getDualFinanceStakingOptionSchema()
+
+  return (
+    <>
+      <Tooltip content="Custom name to identify the Staking Option">
+        <Input
+          label="Name"
+          value={form.soName}
+          type="text"
+          onChange={(evt) =>
+            handleSetForm({
+              value: evt.target.value,
+              propertyName: 'soName',
+            })
+          }
+          error={formErrors['soName']}
+        />
+      </Tooltip>
+      <Tooltip content="Treasury owned account providing the assets for the option. When the recipient exercises, these are the tokens they receive.">
+        <GovernedAccountSelect
+          label="Base Treasury"
+          governedAccounts={governedTokenAccountsWithoutNfts}
+          onChange={(value) => {
+            handleSetForm({ value, propertyName: 'baseTreasury' })
+          }}
+          value={form.baseTreasury}
+          error={formErrors['baseTreasury']}
+          shouldBeGoverned={shouldBeGoverned}
+          governance={governance}
+        ></GovernedAccountSelect>
+      </Tooltip>
+      <Tooltip content="Treasury owned account receiving payment for the option exercise. This is where payments from exercise accumulate.">
+        <GovernedAccountSelect
+          label="Quote Treasury"
+          governedAccounts={governedTokenAccountsWithoutNfts}
+          onChange={(value) => {
+            handleSetForm({ value, propertyName: 'quoteTreasury' })
+          }}
+          value={form.quoteTreasury}
+          error={formErrors['quoteTreasury']}
+          shouldBeGoverned={shouldBeGoverned}
+          governance={governance}
+        ></GovernedAccountSelect>
+      </Tooltip>
+      <Tooltip content="How many tokens are in the staking options. Units are in atoms of the base token.">
+        <Input
+          label="Quantity"
+          value={form.numTokens}
+          type="number"
+          onChange={(evt) =>
+            handleSetForm({
+              value: evt.target.value,
+              propertyName: 'numTokens',
+            })
+          }
+          error={formErrors['numTokens']}
+        />
+      </Tooltip>
+      <Tooltip content="Date in unix seconds for option expiration">
+        <Input
+          label="Expiration"
+          value={form.optionExpirationUnixSeconds}
+          type="number"
+          onChange={(evt) =>
+            handleSetForm({
+              value: evt.target.value,
+              propertyName: 'optionExpirationUnixSeconds',
+            })
+          }
+          error={formErrors['optionExpirationUnixSeconds']}
+        />
+      </Tooltip>
+      <Tooltip content="Strike price for the staking option. Units are atomic quote atoms per lot">
```

Comment:
Clarify what to do for puts by inverting the strike in USDC terms

Reply:
Already says that, but added an example to be more clear